lib/addons/container-insights/index.ts

```diff
@@ -32,27 +29,66 @@ export class ContainerInsightsAddOn extends HelmAddOn {
     /**
      * @override
      */
+    @conflictsWith("AdotCollectorAddOn")
     deploy(clusterInfo: ClusterInfo): Promise<Construct> {
         const cluster = clusterInfo.cluster;        
         const nodeGroups = assertEC2NodeGroup(clusterInfo, ContainerInsightsAddOn.name);
-
         const policy = ManagedPolicy.fromAwsManagedPolicyName('CloudWatchAgentServerPolicy');
         
         nodeGroups.forEach(nodeGroup => {
             nodeGroup.role.addManagedPolicy(policy);
         });
 
+        // Create an adot-collector service account.
+        const serviceAccountName = "adot-collector-sa";
+        let serviceAccountNamespace;
+
+        if (this.props.namespace) {
+            serviceAccountNamespace = this.props.namespace;
+        }
+        else {
+            serviceAccountNamespace = "amazon-metrics"
```

Comment:
Missing `;`
applies elsewhere (we will need to apply a better linter however I believe make lint should catch it)